packages/kobotoolbox/src/http.js

```diff
@@ -0,0 +1,138 @@
+import { expandReferences } from '@openfn/language-common/util';
+import * as util from './Utils';
+
+/**
+ * Options object
+ * @typedef {Object} RequestOptions
+ * @property {object} query - An object of query parameters to be encoded into the URL
+ * @property {object} headers - An object of all request headers
+ * @property {string} [parseAs='json'] - The response format to parse (e.g., 'json', 'text', or 'stream')
+ */
+
+/**
+ * Make a GET request to any Kobotoolbox endpoint.
+ * @public
+ * @function
+ * @example <caption>GET forms resource</caption>
+ * http.get(
+ *  "/assets/",
+ *  {
+ *    query: {
+ *      format: json
```

Comment:
Let's default the format to JSON so that user's don't need to do this :) We ought to document this too (ie, on the description we should add `Responses will be returned in JSON format` or something)